dotcom-rendering/src/client/main.web.ts

```diff
@@ -94,6 +94,15 @@ void (async () => {
 		},
 	);
 
+	void startup(
+		'userFeatures',
+		() =>
+			import(
+				/* webpackMode: 'eager' */ './userFeatures/user-features'
+			).then(({ refresh }) => refresh()),
+		{ priority: 'critical' },
+	);
+
```

Comment:
It would be good to consider whether this is really `critical` or whether it should be a `feature`.

As [mentioned previously](https://github.com/guardian/dotcom-rendering/pull/9345#discussion_r1374641756), this leads to a 20% increase in the size of the initial bundle, and it would be good to know whether there are any further mitigations steps that will be taken.

Reply:
This feature is critical in our opinion, as it sets the user subscription and ad free cookie, needed to provide the appropriate user experience. We can follow up with improving the code size.
Historically it ran as part of the `commercial` code base, but it should be part of the `dcr` platform, to be supported by supporter revenue.